godal.cpp

```diff
@@ -1063,6 +1074,15 @@ void godalGeometryTransform(cctx *ctx, OGRGeometryH geom, OGRCoordinateTransform
 	godalUnwrap();
 }
 
+void godalLayerCopyFeature(cctx *ctx, OGRLayerH layer, OGRFeatureH feat) {
+	godalWrap(ctx);
+	OGRErr oe = OGR_L_CreateFeature(layer,feat);
```

Comment:
there's an inconsistency here: godalLayerCopyFeature vs OGR_L_CreateFeature

Reply:
I did hesitate how to name the go & cpp functions as there is already a "NewFeature" function to create new feature from a geometry.
Do you have any suggestion ?

Reply:
yes, the vector api is messy :(
I would use CreateFeature everywhere, i.e. Layer.CreateFeature for the go api, and godalLayerCreateFeature in godal.cpp